packages/now-static-build/tsconfig.json

```diff
@@ -11,11 +11,10 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "outDir": "dist",
+    "resolveJsonModule": true,
```

Comment:
Can we avoid using `resolveJsonModule`? It caused issues with #3315 that were not detectible with our tests (it made npm publish miss certain files).

Reply:
I updated but I didn't see any difference in the way dist files were compiled between require and import. Both resulted in inlining `package.json` in `dist/index.js`.

Reply:
It might have something to do with the tests written in TS vs JS 🤷‍♂ 

Better safe than sorry I suppose. I'll give this one more look and maybe we can get a second reviewing just to be sure.